apis/r/tools/get_tarball.R

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env Rscript
 
 ## version pinning info
-tiledb_core_version <- "2.19.0"
-# 8-nybble hash for 2.19.0 only. Please see https://github.com/TileDB-Inc/TileDB/pull/4599.
-tiledb_core_sha1 <- "fa30a88a"
+tiledb_core_version <- "2.19.1"
+# 8-nybble hash for 2.19.0 and 2.19.1 only. Please see https://github.com/TileDB-Inc/TileDB/pull/4599.
+tiledb_core_sha1 <- "29ceb3e7"
 
 if ( ! dir.exists("inst/") ) {
     stop("No 'inst/' directory. Exiting.", call. = FALSE)
 }
 
 makeUrl <- function(arch, ver=tiledb_core_version, sha1=tiledb_core_sha1) {
-    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/%s/tiledb-%s-%s-%s.tar.gz", ver, arch, ver, sha1)
+-    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/%s/tiledb-%s-%s-%s.tar.gz", ver, arch, ver, sha1)
+    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/2.19.1/tiledb-2.19.1-2.19.1-2.19.1.tar.gz", ver, arch, ver, sha1)
```

Comment:
You may have left one line.

Reply:
🙏

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/2084/commits/ee2c8ccb12a08db218aaa1e5b0a9a9feae98ae15

Reply:
@eddelbuettel resolved

Reply:
`scripts/update-tiledb-version.py i.j.k` _always_ requires a manual edit on that file -- which I had left half-done, and which you spotted (thank you)